.github/workflows/ci.yml

```diff
@@ -230,7 +235,7 @@ jobs:
         run: invoke ci-tests -m "unit" --xdist --timeout=1.5 --slowest=8
 
   cloud-tests:
-    if: github.event.pull_request.draft == false && github.event.pull_request.head.repo.full_name == github.repository
+    if: github.event.pull_request.draft == false && (github.event.pull_request.head.repo.full_name == github.repository || github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' )
```

Comment:
The same pattern (`${{env.RUN_ON_PR_SCHEDULE_AND_TRIGGER}}`)didn't work here. Would like to merge and unblock first and improve iteratively